README.md

```diff
@@ -88,6 +88,7 @@ According to the [CPC Charter](https://github.com/openjs-foundation/cross-projec
 - **Mocha**: Christopher Hiller (@boneskull)
 - **Node-RED**: Nick O'Leary (@knolleary)
 - **webhint**: Antón Molleda (@molant)
+- **WebdriverIO**: Christian Bromann (@christian-bromann)
```

Comment:
This is not a list of projects, but a list of voting members of the CPC that represent growth & at-large project (see the preceding paragraph).

Reply:
To clarify, these representatives were chosen back in May. Next spring we'll organise another selection, and should only end up with two representatives in this category.